#!/usr/bin/env python

PACKAGE="cob_keyframe_detector"
#import roslib; roslib.load_manifest(PACKAGE)

from math import pi

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("r_limit", double_t, 4096, "COMMENT", 0.017, -3.14, 3.14)
gen.add("p_limit", double_t, 8192, "COMMENT", 0.017, -3.14, 3.14)
gen.add("y_limit", double_t, 16384, "COMMENT", 0.017, -3.14, 3.14)
gen.add("distance_limit", double_t, 32768, "limit of the distance", 0.025, 0, 1)
gen.add("trigger_always", bool_t, 32768, "should trigger always", False)

exit(gen.generate(PACKAGE, "dynamic_reconfigure_node", "keyframe_detector"))
